linode/manager
# refactor: [M3-7720] - Refactor AccessKeyTable - Eliminate react anti patterns

## Description 📝

- Refactors Object-Storage - AccessKeyTable component.
- Eliminates react anti patterns like calling functions that render JSX.
- No Visual changes.


## How to test 🧪
(How to verify changes)
- Validate Object storage Access key flow - create/update/revoke.
- Validate the Access Key landing page and ensure there are no regressions in the Access Key flow.

## As an Author I have considered 🤔

*Check all that apply*
